frontend/src/components/simple-auto-filter/simple-auto-filter.tsx

```diff
@@ -80,7 +90,64 @@ const SimpleAutoFilter: FC = () => {
     navigate(API.SEARCH);
   };
 
-  if (isLoading) return <h1>Loading...</h1>;
+  const brandsOptions = useMemo(
+    () =>
+      brands?.map(
+        (item) =>
+          ({
+            label: item.name,
+            id: item.id,
+          } as AutocompleteValueType),
+      ),
+    [brands],
+  );
+
+  const modelsOptions = useMemo(
+    () =>
+      models?.map(
+        (item) =>
+          ({
+            label: item.name,
+            id: item.id,
+          } as AutocompleteValueType),
+      ),
+
+    [models],
+  );
+
+  const selectedBrandName = useMemo(() => {
+    return getValueById(brands || [], brandId);
+  }, [brandDetails[0]]);
```

Comment:
Please extract brandDetails[0] into a variable - `const [smth] = brandDetails;`

Reply:
That's already done on line 35, just forgot use variables in `deps`